blobfuse/src/DataLakeBfsClient.cpp

```diff
@@ -40,6 +40,10 @@ bool DataLakeBfsClient::AuthenticateStorage()
         
     if(m_adls_client != NULL)
     {
+        if (m_adls_client->get_blob_client() && m_adls_client->get_blob_client()->context())
```

Comment:
this is redundant. The line above is doing it. So remove this one

Reply:
This is not redundant. In case of ADLS we have a blob client object created explicitly and one blob clien tobject adls object creates internally. For the external one we were always setting the exponential retry policy however the internal blob client of adls object was missed. This was the reason for not having exponential retry only in certain flows. With this line we are setting the retry policy for implicit blob client object created inside adls object. Ref : m_adls_client->get_blob_client()